provider/win_driver_installer_test.py

```diff
@@ -269,270 +268,16 @@ def iozone_test(test, params, vm, images):
-    if installed:
-        test.log.info("Winfsp tool is already installed.")
-    else:
-        install_cmd = utils_misc.set_winutils_letter(session, install_cmd)
-        session.cmd(install_cmd, cmd_timeout)
-        if not utils_misc.wait_for(lambda: not session.cmd_status(
-                check_installed_cmd), 60):
-            test.error("Winfsp tool is not installed.")
-
-
-def get_virtiofs_driver_letter(test, fs_target, session):
-    """
-    Get the virtiofs driver letter( Windows guest only )
-    :param test: QEMU test object
-    :param fs_target: virtio fs target
-    :param session: the session of guest
-    :return driver_letter: the driver letter of the virtiofs
-    """
-    test.log.info("Get driver letter of virtio fs target, the driver label is "
-                  "%s." % fs_target)
-    vol_con = "VolumeName='%s'" % fs_target
-    driver_letter = utils_misc.wait_for(
-        lambda: utils_misc.get_win_disk_vol(session, condition=vol_con),
-        timeout=120)
-    if driver_letter is None:
-        test.fail("Could not get virtio-fs mounted driver letter.")
-    return driver_letter
-
-
 def viofs_basic_io_test(test, params, vm):
     """
     Basic io test in Windows guest
     :param test: QEMU test object
     :param params: Dictionary with the test parameters
     :param vm: the vm object
     """
-    error_context.context("Running viofs basic io test", LOG_JOB.info)
     session = vm.wait_for_login()
-    test_file = params.get('test_file', "virtio_fs_test_file")
-    cmd_dd = params.get("cmd_dd", 'dd if=/dev/random of=%s bs=1M count=200')
-    io_timeout = params.get("timeout", 120)
-    cmd_md5 = params.get("cmd_md5", '%s: && md5sum.exe %s')
-    fs_source = params.get("fs_source_dir")
-    base_dir = params.get("fs_source_base_dir",
-                          data_dir.get_data_dir())
-    if not os.path.isabs(fs_source):
-        fs_source = os.path.join(base_dir, fs_source)
-
-    host_data = os.path.join(fs_source, test_file)
-
-    fs_target = params.get("fs_target")
-    driver_letter = get_virtiofs_driver_letter(test, fs_target, session)
-    fs_dest = "%s:" % driver_letter
-    guest_file = os.path.join(fs_dest, test_file)
-
-    test.log.info("Creating file under %s inside guest." % fs_dest)
-    session.cmd(cmd_dd % guest_file, io_timeout)
-
-    guest_file_win = guest_file.replace("/", "\\")
-    cmd_md5_vm = cmd_md5 % (driver_letter, guest_file_win)
-    md5_guest = session.cmd_output(cmd_md5_vm, io_timeout).strip().split()[0]
-
-    test.log.info("md5 of the guest file: " + md5_guest)
-    md5_host = process.run("md5sum %s" % host_data,
-                           io_timeout).stdout_text.strip().split()[0]
-    test.log.info("md5 of the host file: " + md5_host)
-    if md5_guest != md5_host:
-        test.fail('The md5 value of host is not same to guest.')
-    else:
-        test.log.info("The md5 of host is as same as md5 of guest.")
+    virtio_fs_utils.basic_io_test(test, params, session)
```

Comment:
cmd_dd in virtio_fs_utils.basic_io_test  is only  suitable for linux guest, pls update them.